portal/code/cx-portal/src/components/pages/Appstore/index.tsx

```diff
@@ -18,19 +17,31 @@ const Appstore = () => {
 
   return (
     <main className="Appstore">
-      <input
+      <SearchInput
```

Comment:
This component gives me an error of theme related icons usage : index.tsx:10 Uncaught TypeError: Cannot destructure property 'icon01' of 'r.palette.icon' as it is undefined.

I think we need to state SharedThemeProvider HOC at very top level to avoid theme related style usage at shared components